frontend/src/pages/offers/PalletsEditor.tsx

```diff
@@ -57,30 +66,95 @@ const PalletsEditor: FunctionComponent<Props> = ({ pallets }) => {
     })
   }
 
+  const [
```

Comment:
The pallet creation code was moved over from the `ViewOfferPage`. There are no functional changes.